front/src/common/Map/MapKey.tsx

```diff
@@ -38,8 +39,8 @@ const MapSettings = ({ toggleMapKey }: MapKeyProps) => {
   ));
 
   return (
-    <div className="map-modal map-modal-dark active">
-      <div className="h2 text-light">{t('map-key:keyTitle')}</div>
+    <div className="map-modal map-modal-dark">
+      <HearderPopUp onClick={toggleMapKey} title={t('map-key:keyTitle')} isLight />
```

Comment:
It would be clearer if the function name was closeMapKeyPopUp

Reply:
Done